scripts/benchmarks_ci.py

```diff
@@ -31,6 +31,7 @@
 from performance.logger import setup_loggers
 from performance.constants import UPLOAD_CONTAINER, UPLOAD_STORAGE_URI, UPLOAD_TOKEN_VAR, UPLOAD_QUEUE
 from channel_map import ChannelMap
+from shared.util import runninginlab
```

Comment:
This won't work as `shared` won't be in the scope for this script. You either need to move this to `common.py` (and fixup uses of it in other places) or something similar.